ios/src/TGGeoPoint.h

```diff
@@ -0,0 +1,27 @@
+//
+//  TGGeoPoint.h
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#ifndef TGGeoPoint_h
+#define TGGeoPoint_h
```

Comment:
FWIW, this is iOS specific code, but even on Mac, LLVM/Clang is the standard compiler toolchain and pragma once is supported. GCC is "supported" as well, but not installed anymore by xcode, which also supports pragma once. So, your portability requirements are limited.

That said, if you look at CoreFoundation code, it's littered with `#ifndef` statements that I'm assuming predate GCC support of pragma once, and definetly predate clang. So, using it is not without precedent, but I wouldn't concern yourself highly with portability in this particular instance.